packages/@aws-cdk/aws-docdb/lib/cluster.ts

```diff
@@ -234,6 +234,74 @@ abstract class DatabaseClusterBase extends Resource implements IDatabaseCluster
   }
 }
 
+/**
+ * Represents an imported database cluster.
+ */
+class ImportedDatabaseCluster extends DatabaseClusterBase implements IDatabaseCluster {
```

Comment:
Out of curiosity - why did you move this class from where it was defined?

Let's maybe leave it where it is - this way, the diff will be significantly smaller?

Reply:
Thanks all for the lighting fast response!
I have made changes based on the following code.
https://github.com/aws/aws-cdk/blob/4071d9b0b6af241e1d91d6e5b828334f1693b8a5/packages/%40aws-cdk/aws-rds/lib/cluster.ts#L427-L487

but I agree with your suggestion. So I left it as is and made fix.